index.html

```diff
@@ -0,0 +1,23 @@
+<html>
+  <head>
+    <title>Photo Carousel</title>
+    <script>
+      class PhotoCarousel extends HTMLElement {
```

Comment:
We need to speak about OOP here. 
Going to create a ticket for that to not lose the focus on it.

Reply:
Ticket has been created https://github.com/Nantiopa/xmundo/issues/2